src/Symfony/Bundle/WebProfilerBundle/Twig/Node/HtmlDebugEnterComment.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\WebProfilerBundle\Twig\Node;
+
+use Twig\Node\Node;
+
+/**
+ * @author Yonel Ceruto <yonelceruto@gmail.com>
+ */
+class HtmlDebugEnterComment extends Node implements \Twig_NodeOutputInterface
```

Comment:
Symfony uses the namespaced version of all Twig classes and interfaces, to make sure we are ready for the future version of Twig dropping support for the non-namespaced version.

Reply:
Yeah, should `Twig\Node\NodeOutputInterface` be used instead.